src/commands/custom/updatemap.js

```diff
@@ -0,0 +1,74 @@
+const { SlashCommandBuilder, PermissionsBitField } = require("discord.js");
+const { loadReactionMap, gitCommit, reactionMapPath } = require("../../utils/reactionutils");
+const fs = require("fs");
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName("updatemap")
+        .setDescription("Update the reaction map")
+        .addStringOption((option) =>
+            option
+                .setName("keyword")
+                .setDescription("The keyword to update")
+                .setRequired(true)
+        )
+        .addStringOption((option) =>
+            option
+                .setName("emoji")
+                .setDescription("The emoji to associate with the keyword")
+                .setRequired(true)
+        ),
+    async execute(interaction) {
+        const keyword = interaction.options.getString("keyword");
+        const emoji = interaction.options.getString("emoji");
+
+        // Check for admin permissions
+        if (
+            !interaction.member.permissions.has(
+                PermissionsBitField.Flags.Administrator
+            )
+        ) {
+            return interaction.reply({
+                content: "You do not have permission to use this command.",
+                ephemeral: true,
+            });
+        }
```

Comment:
i dont want to sound like a broken record but uh the thing with the flag and the default and the /clear as an example